pontoon/contributors/views.py

```diff
@@ -310,24 +310,30 @@ def settings(request):
     all_locales = list(Locale.objects.all())
     all_locales.insert(0, default_homepage_locale)
 
-    # Set custom homepage selector value
-    custom_homepage_locale = profile.custom_homepage
-    if custom_homepage_locale:
+    # Set default for custom homepage locale based on name
+    custom_homepage_locale_object = get_locale_or_redirect(profile.custom_homepage)
```

Comment:
I would just leave the name `custom_homepage_locale`.

Reply:
Also, you should first check if we have `profile.custom_homepage`.

If we don't, use `default_homepage_locale`.
Only if we do, call `get_locale_or_redirect()`, otherwise it will fail (redirect).